google/cloud/bigtable/tests/data_integration_test.cc

```diff
@@ -587,3 +587,50 @@ TEST_F(DataIntegrationTest, TableSampleRowKeysTest) {
   auto last = samples.back();
   EXPECT_LT(0, last.offset_bytes);
 }
+
+TEST_F(DataIntegrationTest, TableReadMultipleCellsBigValue) {
+  if (UsingCloudBigtableEmulator()) {
+    // TODO(#151) - remove workarounds for emulator bug(s).
+    return;
+  }
+
+  std::string const table_id = RandomTableId();
+  auto table = CreateTable(table_id, table_config);
+
+  std::string const row_key = "row-key-1";
+  // cell vector contains 8 cell of ~32 MiB
+  auto const MiB = 1024L * 1024L;
+
+  std::string value((32 * MiB) - 1024, 'a');
```

Comment:
Can you add a comment explaining why you subtract 1024 here?

Reply:
the subtraction of 1024 is to offset the value of the cell. I am try to get the row size to 256 MiB which will be 8 cell of 32 MiB  but when when read the row it exceeded the max limit of 256 MiB.
it total size goes a few bytes above.

Reply:
That makes sense, can you put a comment explaining this? Something like:

```C++
// We want to create a message of around 256MiB, if we create a row with *exactly* 256MiB the
// message overhead exceeds the 256MiB limit, so make the cells slightly smaller.
```